lldb/include/lldb/Utility/FileSpecList.h

```diff
@@ -45,13 +48,61 @@ class FileSpecHolder {
   }
 };
 
+class SupportFileList {
```

Comment:
Let's call this `SupportFile` to match the `SupportFileList`. This is where I'll add my MD5 checksum to.